modules/tinymce/src/plugins/link/main/ts/api/Commands.ts

```diff
@@ -1,17 +1,20 @@
 import Editor from 'tinymce/core/api/Editor';
 
-import * as Actions from '../core/Actions';
+import * as Dialog from '../ui/Dialog';
 import * as Options from './Options';
 
+interface Dialog {
+  readonly 'dialog': boolean;
```

Comment:
```suggestion
  readonly dialog?: boolean;
```
There shouldn't be quotes here and I'm not convinced this should be mandatory? It would lock us in again in future.